.github/workflows/server-prod-deploy.yml

```diff
@@ -0,0 +1,120 @@
+name: Deploy production server
+
+on:
+    push:
+        branches:
+            - deploy-prod
```

Comment:
We could probably change this to master or add master to the triggers. We can then try merging #133 for the next prod deployment.

Reply:
Hmm, if we're gonna take the opportunity to rearrange things, do you want to just make the branches 1:1 with their environments? I think that makes sense. 

`staging` branch deploys to `staging` -- maybe `staging` replaces dev? I think that makes a lot of sense potentially.
`prod` branch deploys to `prod`

Reply:
I like staging and prod, and I agree that staging replacing dev would possibly be a better workflow since merging dev into staging is a sort of pointless extra step each time.

Reply:
Alright, I updated all the files I think. Notably, docs I kept only to staging only. I think we'll only have docs deploy off of staging since that will have the most up-to-date dev stuff.